build-bin/dist_to_gh_pages

```diff
@@ -15,26 +15,37 @@
 
 set -ue
 
+ORIGIN_BRANCH='master'
+TARGET_BRANCH='gh-pages'
+
 # Update gh-pages
-git fetch origin gh-pages:gh-pages
-git checkout gh-pages
```

Comment:
let me know how you tested this script as historically changes to scripts like this break and I need to fix them later when deployment failed.